addons/website_slides/controllers/main.py

```diff
@@ -797,6 +797,26 @@ def create_slide(self, *args, **post):
             else:
                 values['category_id'] = post['category_id'][0]
 
+        new_certification = False
+        if post.get('survey_id'):
+            if post['survey_id'][0] == 0:
+                survey_id = request.env['survey.survey'].create({
```

Comment:
Side / Functional question: is there any possibilities to ask for a badge for the new-ly created survey ? @dbeguin has a task to automatically create badges for some certification surveys, maybe it could be integrated here so that people can create a survey with a badge easily ... to check with Sbu probably.

Reply:
Should merge mine first then but yes, good idea.